### PR TITLE
Use azion-framework-adapter npm package

### PR DESCRIPTION
**Pull request by MagnunAVFAzion**
Use azion-framework-adapte from npmjs registry instead of clone and build the github project.

The `npx` use will not force the user to install the depency globally.

Also add initialization check in build and publish commands.

### CODE REVIEW

File: webdev/flareact/webdev.sh [hunk 1]
```diff
@@ -104,23 +85,22 @@ fi
...
         mkdir -p public ;;
 
     build )
+        check_init || exit $?
         check_envvars || exit $?
-        check_azion_framework_adapter || exit $?
         echo "{}" > ./args.json
 
-        azion-framework-adapter build --config ./azion/kv.json || exit $?;;
+        npx --yes azion-framework-adapter@0.2.0 build --config ./azion/kv.json || exit $?;;
```
Comment (lmalheiro) [inline webdev/flareact/webdev.sh:98]: Wouldn't be better to drop the path number?
```suggestion
        npx --yes azion-framework-adapter@0.2 build --config ./azion/kv.json || exit $?;;
```

File: webdev/flareact/webdev.sh [hunk 2]
```diff
@@ -104,23 +85,22 @@ fi
...
 
     publish )
+        check_init || exit $?
         check_envvars || exit $?
-        check_azion_framework_adapter || exit $?
 
         # Publish only assets
-        azion-framework-adapter publish -s --config ./azion/kv.json || exit $?;;
+        npx --yes azion-framework-adapter@0.2.0 publish -s --config ./azion/kv.json || exit $?;;
```
Comment (lmalheiro) [inline webdev/flareact/webdev.sh:105]: Idem as above.
```suggestion
        npx --yes azion-framework-adapter@0.2 publish -s --config ./azion/kv.json || exit $?;;
```

File: webdev/nextjs/webdev.sh [hunk 3]
```diff
@@ -157,15 +138,15 @@ case "$1" in
         npx next build  || exit $?
         npx next export || exit $?
         cd azion/cells-site-template || exit $?
-        azion-framework-adapter build --config ../kv.json \
+        npx --yes azion-framework-adapter@0.2.0 build --config ../kv.json \
```
Comment (lmalheiro) [inline webdev/nextjs/webdev.sh:141]:
```suggestion
        npx --yes azion-framework-adapter@0.2 build --config ../kv.json \
```
Dropping the patch number should allow automatic to update the npm package  for bugs.

File: webdev/nextjs/webdev.sh [hunk 4]
```diff
@@ -157,15 +138,15 @@ case "$1" in
...
 
     publish )
+        check_init || exit $?
         check_envvars || exit $?
-        check_azion_framework_adapter || exit $?
 
         cd azion/cells-site-template || exit $?
         # Publish only assets
-        azion-framework-adapter publish --config ../kv.json \
+        npx --yes azion-framework-adapter@0.2.0 publish --config ../kv.json \
```
Comment (lmalheiro) [inline webdev/nextjs/webdev.sh:150]:
```suggestion
        npx --yes azion-framework-adapter@0.2.0 publish --config ../kv.json \
```
Dropping the patch number should allow automatic to update the npm package  for bugs.

**Review by lmalheiro — APPROVED** [review]

**Review by PatrickMenoti — APPROVED** [review]
lgtm

File: webdev/flareact/webdev.sh (hunk 1 above)
Comment (MagnunAVFAzion) [reply webdev/flareact/webdev.sh:98]: npx creates a cache in local machine (in ~/.npm/_npx) and doesn't check package newer versions. To solve this we have to use `npx PACKAGE@latest` or `npx PACKAGE@x.y.z`. 
If we use without PATCH user could not receive new updates. 
We chose to use x.y.z to have more secure deployments.

References:
- https://github.com/npm/cli/issues/2329
- https://github.com/npm/cli/issues/4108

File: webdev/flareact/webdev.sh (hunk 2 above)
Comment (MagnunAVFAzion) [reply webdev/flareact/webdev.sh:105]: https://github.com/aziontech/azioncli-template/pull/35#discussion_r1001968121

File: webdev/nextjs/webdev.sh (hunk 3 above)
Comment (MagnunAVFAzion) [reply webdev/nextjs/webdev.sh:141]: https://github.com/aziontech/azioncli-template/pull/35#discussion_r1001968121

File: webdev/nextjs/webdev.sh (hunk 4 above)
Comment (MagnunAVFAzion) [reply webdev/nextjs/webdev.sh:150]: https://github.com/aziontech/azioncli-template/pull/35#discussion_r1001968121

**Pull request merged**